### PR TITLE
Add Pocket's TOS and Privacy Policy docs - initial import

### PR DESCRIPTION
**Pull request by stevejalim**
This changeset brings over Pocket's two legal pages as markdown, extracted from the HTML of the current live pages.

These will be pulled into bedrock as part of the regular sync and will be displayed when Bedrock operates in Pocket Mode, to serve the Pocket marketing pages

Contributes to https://github.com/mozilla/bedrock/issues/11723

### CODE REVIEW

**Comment by stevejalim** [comment]
@craigcook @peiying2 Would you mind taking a look at this initial import and let me know what's worth tuning at this stage? Thanks very much!

File: en/pocket_tos.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+## ACCESS BY CHILDREN
+
+In the United States, if you are a device manufacturer or other third party (the "Sponsor") that provides devices to children under the age of 13 that utilize the Pocket Technologies, you assume the responsibility for complying with the U.S. Children’s Online Privacy Protection Act ("COPPA") and, to the extent applicable, The Family Educational Rights and Privacy Act ("FERPA"). This means that the Sponsor must notify parents/guardians of the information to be collected and obtain parental/guardian consent before collecting and sharing with the Pocket application the personal information of children under the age of 13 in order to establish an account or use the Pocket Technologies. Schools may under appropriate circumstances provide such consent on behalf of parents/guardians. For more information on complying with COPPA, see the Federal Trade Commission’s website at [https://www.ftc.gov/privacy/coppafaqs.shtm](https://www.ftc.gov/privacy/coppafaqs.shtm).
```
Comment (stevejalim) [inline en/pocket_tos.md:167]: This ftc.gov URL currently 404s. We need to find an alternative (I've asked the Pocket team if https://www.ftc.gov/business-guidance/resources/complying-coppa-frequently-asked-questions might work)

File: en/pocket_privacy_policy.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+* [Sharing of Information](#sharing)
+* [Keeping Information](#keeping)
+* [Transfer of Information](#transfer)
+* [Security](#security)
+* [Phishing](#phishing)
+* [Managing My Privacy and Email Settings](#opt-out)
+* [Amendments to Policy](#amendments)
+* [Contact](#contact)
+
+## USER ACCOUNTS AND COLLECTION OF INFORMATION
```
Comment (flodolo) [inline en/pocket_privacy_policy.md:24]: Titles use sentence case in existing docs, I think this should do to the same.

File: en/pocket_privacy_policy.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+We understand your concerns about privacy. We want you to understand what types of information we collect and what may happen to that information if you install the Pocket application, visit the Pocket Website or use any of the Pocket Technologies.
+
+Our Privacy Policy includes the following provisions, and you can jump to each by selecting the links below:
+
+* [User Accounts and Collection of Information](#accounts)
```
Comment (flodolo) [inline en/pocket_privacy_policy.md:11]: You have links to anchors but no anchors? There should be something like `{: #accounts }` next to the reference title.

https://raw.githubusercontent.com/mozilla/legal-docs/main/en/firefox_privacy_notice.md

File: en/pocket_privacy_policy.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
+# Privacy Policy
+
+## *Posted October 11, 2021*
```
Comment (flodolo) [inline en/pocket_privacy_policy.md:3]: Other documents don't use an heading level for this, and have a datetime directive. Also, the language is different.

```
*Effective October 11, 2021*
{: datetime="2021-10-11" }

File: en/pocket_privacy_policy.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+### Your Choices
+
+* Cookie Preference tool: By default, Pocket sets necessary and analytics cookies, and disables personalized advertising cookies. You can manage these cookies from this Cookie Preferences tool. [Learn more](https://help.getpocket.com/article/1163-pocket-website-cookies-faq).
+* Your browser: Your browser preferences menu can help you manage cookies and prevent cross-site tracking.
+* Google’s Analytics opt-out: Install the [Google Analytics Opt-out Browser Add-on](https://tools.google.com/dlpage/gaoptout), which prohibits data transmission to Google Analytics.
+* External opt-out tools: Some third- party organizations provide ways of opting out of personalized advertising delivered by participating companies. For information about how to opt-out, you can visit these organizations’ websites: [Network Advertising Initiative’s opt-out tools](https://optout.networkadvertising.org/?c=1), the [Digital Advertising Initiative’s Browser Check tool](https://optout.aboutads.info/?c=2&lang=EN), and the [European Interactive Digital Advertising Alliance Your Ad Choices](https://www.youronlinechoices.com/uk/your-ad-choices).
```
Comment (flodolo) [inline en/pocket_privacy_policy.md:51]: The second link should probably go without `lang=EN`, it's unfortunate that the third forces you to select a country.

File: en/pocket_tos.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+
+## COPYRIGHT NOTICE
+
+Copyright © 2012-2021 Mozilla Corporation All rights reserved.
```
Comment (flodolo) [inline en/pocket_tos.md:181]: I don't think I've seen a © on Mozilla legal-docs before…

**Review by stevejalim — COMMENTED** [review]
Thanks for the review and comments @flodolo - I've addressed them all and made appropriate amendments. Let me know what you think.

File: en/pocket_privacy_policy.md (hunk 3 above)
Comment (flodolo) [reply en/pocket_privacy_policy.md:12]: Sounds good to me.

File: en/pocket_privacy_policy.md (hunk 5 above)
Comment (flodolo) [reply en/pocket_privacy_policy.md]: I thought about it, but that website is confusing. From the home page, I had no idea where to click next, so it's probably safer to keep the link as is.

Comment (stevejalim) [reply en/pocket_privacy_policy.md]: Cool - reverted the change to the third link

**Comment by stevejalim** [comment]
@flodolo If you're happy, are you able to approve this PR, or does it need to go to Michael?

Also, is there anything blocking this content reaching the `prod` branch before the end of the month, please?

**Comment by flodolo** [comment]
> @flodolo If you're happy, are you able to approve this PR, or does it need to go to Michael?

I would assume Michael is already aware of these docs, but maybe double check with him?

I assume this doesn't have any impact on mozilla.org (standard mode)? If so, we can merge and push at any time, and make further fixes as needed.

Let me know if you want me to merge.

**Review by flodolo — APPROVED** [review]

**Comment by stevejalim** [comment]
@flodolo yeah, thanks - let's merge this and tweak as we need. I've reached out to Michael via slack re the 404ing link so we can fix that post merge.

Nothing in this changeset will affect mozorg, correct

**Pull request merged**